packages/lab/src/contact-details/ContactDetails.css

```diff
@@ -270,22 +268,56 @@
 }
 
 .uitkContactFavoriteToggle-deselected {
-  fill: var(--contact-details-deselected-icon-color);
+  fill: var(--contactDetails-deselected-icon-color);
 }
 
 .uitkContactFavoriteToggle-selecting {
-  fill: var(--contact-details-selecting-icon-color);
+  fill: var(--contactDetails-selecting-icon-color);
 }
 
 .uitkContactFavoriteToggle-selected {
-  fill: var(--contact-details-selected-icon-color);
+  fill: var(--contactDetails-selected-icon-color);
 }
 
 .uitkContactFavoriteToggle-deselecting {
-  fill: var(--contact-details-deselecting-icon-color);
+  fill: var(--contactDetails-deselecting-icon-color);
 }
 
 .uitkContactFavoriteToggle-svg {
-  height: var(--contact-details-icon-size);
-  width: var(--contact-details-icon-size);
+  height: var(--contactDetails-icon-size);
+  width: var(--contactDetails-icon-size);
+}
+
+/* backwardsCompat */
+
+.uitk-density-touch .backwardsCompat.uitkContactDetails-default .uitkContactPrimaryInfo {
```

Comment:
usually would have backwardsCompat just on the root class name, but this is OK as long as backwards compat class goes on the same div as all of these variants and they're not nested

Reply:
uitkContactDetails-default comes on the same level as uitkContactDetails and will never be present without the main uitkContactDetails class. Was just avoiding a very long css selector